README.md

```diff
@@ -1,45 +1,31 @@
 # Dash Core Components
 
-This package provides the core React component suite for [Dash][].
+This package provides the core React components suite for [Dash][].
 
 [![CircleCI](https://circleci.com/gh/plotly/dash-core-components.svg?style=svg)](https://circleci.com/gh/plotly/dash-core-components)
 
 ## Development
 
 ### Testing Locally
 
-1. Install the dependencies with:
+The `dash` package contains some tools to build components and drive the bundles build process.
+To avoid the circulair dependency situation, we don't add dash as a required install in dash-core-components setup. But in order to do the local development, you need to install `dash` before everything.
```

Comment:
```suggestion
To avoid the circular dependency situation, we don't add `dash` as a required install in the `dash-core-components` setup. But, in order to development locally, you need to install `dash` before everything.
```